Doc/library/importlib.rst

```diff
@@ -1719,6 +1719,27 @@ To import a Python source file directly, use the following recipe
   spec.loader.exec_module(module)
 
 
+Implementing lazy imports
+'''''''''''''''''''''''''
+
+The example below shows how to implement lazy imports::
+
+  import sys
+  import importlib.util
```

Comment:
```suggestion
  import importlib.util

```

Reply:
Just curious, what are you achieving with this edit?

Reply:
oh! I see, you added a new line. Okay makes sense.

Reply:
Not applicable in console mode I guess that is if you meant a new line.